pkg/publicapi/middleware/error_handler.go

```diff
@@ -28,13 +31,17 @@ func CustomHTTPErrorHandler(err error, c echo.Context) {
 		// size accepted
 		code = e.Code
 		message = e.Message.(string)
+		errorCode = string(models.InternalError)
+		component = "Echo"
```

Comment:
`Echo` is an internal implementation details. Should we use `APIServer` or something like that?

Reply:
Makes sense